src/lua/scripts/inspect.lua

```diff
@@ -0,0 +1,333 @@
+local _tl_compat; if (tonumber((_VERSION or ''):match('[%d.]*$')) or 0) < 5.3 then local p, m = pcall(require, 'compat53.module'); if p then _tl_compat = m end end; local math = _tl_compat and _tl_compat.math or math; local string = _tl_compat and _tl_compat.string or string; local table = _tl_compat and _tl_compat.table or table
```

Comment:
Is this code in a single line intentionally?

Reply:
Yes, see https://github.com/kikito/inspect.lua/blob/9c8a68da3033f1393b84b498313572654af82b8f/inspect.lua#L1

Reply:
They also forgot to update their version string 

https://github.com/kikito/inspect.lua/blob/9c8a68da3033f1393b84b498313572654af82b8f/inspect.lua#L20

(should be `3.1.3` now)

Reply:
I'd vote for leaving it like it is. We have the most recent version included.